### PR TITLE
Global Settings - Mountpoint ignore options

### PR DESCRIPTION
**Pull request by SourceDoctor**
Mountpoints to be ignored no configurable via global Settings
image

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: misc/config_definitions.json [hunk 1]
```diff
@@ -3234,10 +3246,16 @@
             "type": "array"
```
Comment (murrant) [inline misc/config_definitions.json]: I think you can validate these are regex with:
```
            "validate": {
                "value.*": "regex"
            }
```

Comment (murrant) [inline misc/config_definitions.json]: I think you can validate these are regex with:
```
            "validate": {
                "value.*": "regex"
            }
```

Comment (murrant) [reply misc/config_definitions.json]: I wish there was a better rule to use...

Comment (murrant) [reply misc/config_definitions.json]: I guess we could define a new validation:
```
        Validator::extend('is_regex', function ($attribute, $value) {
            return @preg_match($value, null) !== false;
        });
```

Comment (SourceDoctor) [reply misc/config_definitions.json]: i only brought it to GUI :-)

Comment (SourceDoctor) [reply misc/config_definitions.json]: > I guess we could define a new validation:
> 
> ```
>         Validator::extend('is_regex', function ($attribute, $value) {
>             return @preg_match($value, null) !== false;
>         }, __(':attribute is not a valid regular expression'));
> ```

do you want me to implement this sample? If so, where?

Comment (murrant) [reply misc/config_definitions.json]: It goes in AppServiceProvider::bootCustomValidators()

Comment (SourceDoctor) [reply misc/config_definitions.json]: ah ok, thx
i'll update file and config definitions in 4-5h. I'm still at work :-)

Comment (SourceDoctor) [reply misc/config_definitions.json]: included :-)

**Comment by SourceDoctor** [comment]
@murrant 
something else ?

**Comment by SourceDoctor** [comment]
@murrant 
something else to do?

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-61-release-changelog-february-2020/11145/1

**Pull request merged**